### DISCUSSION TITLE
Update aplikasi baru

### DISCUSSION THREAD

**Discussion by agungp86** [mas-elkhanza/SIMRS-Khanza#110] [Q&A]
Saya sudah pakai aplikasi ini sejak 2019, sekarang mau update pakai yg terbaru. Aplikasi yang baru bisa saya install dengan normal tapi kendala muncul saat saya import database yg lama ke database yg baru. Bagaimana cara supaya data di database lama bisa masuk langsung ke database yang baru? Terimakasih

**Comment by mas-elkhanza** [comment]
sinkron struktur pakai navicat
